frontend/src/components/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -20,46 +22,50 @@ const SignUpForm: React.FC<Props> = ({ onSubmit }) => {
 
   return (
     <>
-      <div className={styles.signUpLogo}>
-        <img src={logo} width="50" alt="logo" />
+      <div className={styles.signUpWrapper}>
+        <img className={styles.signUpLogo} src={logo} alt="logo" />
+        <h1 className={styles.signUpTitle}>Sign Up</h1>
+        <div className={styles.signInLink}>
+          <span>Already have an account? </span>
+          <Link to={AppRoute.SIGN_IN}>Sign in</Link>
+        </div>
+        <form className={styles.signUpForm} onSubmit={handleSubmit(onSubmit)}>
+          <p className={styles.signUpFormContent}>
```

Comment:
Use div instead.

Reply:
> Use div instead.
Where?

Reply:
instead of `<p></p>` => `<div className={styles.signUpFormContent}></div>`